spring-cloud-gcp-pubsub/pom.xml

```diff
@@ -25,7 +25,10 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-gcp-core</artifactId>
 		</dependency>
-
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
```

Comment:
Should it be `optional`, since it's possible to use the module with Spring Integration?